gql/resolver_sealingpipeline.go

```diff
@@ -124,12 +127,16 @@ type worker struct {
 	Sector int32
 }
 
-type sealingPipelineResolver struct {
+type SealingPipelineState struct {
 	WaitDeals    waitDeals
 	SectorStates sectorStates
 	Workers      []*worker
 }
 
+type sealingPipelineResolver struct {
+	SealingPipelineState
+}
+
```

Comment:
nit: do we actually need the sealingPipelineResolver? Can we just return a sealingPipelineState object from `func (r *resolver) SealingPipeline`?